lib/src/base-styles/_nx-card.scss

```diff
@@ -107,75 +78,64 @@
   }
 
   .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
+    align-self: center;
+    grid-area: card-content;
     margin: 0;
     padding: 0;
+    row-gap: 0;
   }
 
   .nx-card__call-out {
-    -ms-grid-row-align: center;
     align-self: center;
-    -ms-grid-row: 1;
-    -ms-grid-row-span: 2;
-    -ms-grid-column: 1;
     grid-area: card-callout;
-    -ms-grid-column-align: center;
     justify-self: center;
```

Comment:
Doesn't do anything since this element's width is dictated by its content

Reply:
> Doesn't do anything since this element's width is dictated by its content

https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/1a904c393841bca80421915a60e0b8e1d7f4305f